src/Views/Todos/Todos.js

```diff
@@ -0,0 +1,68 @@
+import './Todos.css';
+import React, { useEffect, useState } from 'react';
+// import { useHistory } from 'react-router-dom';
+import { fetchTodos, addTodo, toggleComplete } from '../../services/fetchtodos';
+
+
+export default function Todos() {
+  const [todo, setTodos] = useState([]);
+  const [makeTodo, setMakeTodo] = useState('');
+  const [error, setError] = useState('');
+//   const history = useHistory();
+
+  useEffect(() => {
+
+    const fetchData = async () => {
+      try {
+        const data = await fetchTodos();
+        setTodos(data);
+
+      } catch (error) {
+        setError(error.message);
+      }
+    };
+    fetchData();
+  }, []);
+
+  const handleClick = async (data) => {
+    try {
+      await toggleComplete(data);
```

Comment:
After toggleComplete, we will want to make another fetch call so that our data is displayed on the page without having to refresh the browser.